test/integration/dom.bpmn.elements.test.ts

```diff
@@ -61,6 +63,32 @@ describe('Bpmn Elements registry - retrieve BPMN elements', () => {
         expectSequenceFlowBpmnElement(bpmnElements[1], { id: 'Flow_2', source: 'Activity_1', target: 'EndEvent_1' });
       });
 
+      it('Pass existing link intermediate event ids', () => {
+        bpmnVisualization.load(readFileSync('../fixtures/bpmn/registry/1-pool-3-lanes-message-start-end-intermediate-events.bpmn'));
+        const bpmnElements = bpmnVisualization.bpmnElementsRegistry.getElementsByIds(['Event_1wihmdr', 'Event_1q818hp']);
+        expect(bpmnElements).toHaveLength(2);
+
+        expectIntermediateCatchEventBpmnElement(bpmnElements[0], {
+          id: 'Event_1wihmdr',
```

Comment:
**nitpick:** please sort properties in alphabetical order for consistency with other tests

Reply:
Will be eventually done with #2742